keyboards/rubi/rubi.c

```diff
@@ -73,7 +73,7 @@ bool led_update_kb(led_t led_state) {
     return true;
 }
 
-__attribute__ ((weak)) void encoder_update_user(uint8_t index, bool clockwise) {
+__attribute__ ((weak)) bool encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
```suggestion
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
```

Reply:
Probably better to move the `_user` to the keymap and leave it out of the keyboard entirely.

Reply:
Moving to user keymap.